src/App.tsx

```diff
@@ -15,36 +17,92 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App: React.FC = () => {
+enum SortType {
```

Comment:
make sure to add default value to enum to use it in code

> Use enum SortType instead of string literals for possible sorting options (including the default one)